### PR TITLE
Updates to Dockerfile, docker-compose.yml

### PR DESCRIPTION
**Pull request by mfrost**
Ended up rewriting a lot of the Docker setup. I would still like to figure out how to move the `node_modules` out of the project directory so we don't have to deal with shuffling those packages around in the volume.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,13 +8,12 @@ RUN curl -sL https://deb.nodesource.com/setup_8.x | bash &&\
...
 RUN groupadd -g ${GROUP_ID} developers &&\
     useradd -l -u ${USER_ID} -g ${GROUP_ID} developer &&\
-    install -d -m 0755 -o developer -g developers /home/developer
-
-RUN mkdir /helios && chown developer:developers /helios
-RUN mkdir /helios/secure && chown developer:developers /helios/secure
-RUN mkdir /helios/tmp && chown developer:developers /helios/tmp
+    install -d -m 0755 -o developer -g developers /home/developer &&\
+    mkdir -p /helios/secure /helios/tmp &&\
+    chown -R developer:developers /helios
```
Comment (cbarber) [inline Dockerfile]: Should we also add:
```
      args:
        USER_ID: $(id -u ${USER})
        GROUP_ID: $(id -g ${USER})
``` 
to docker-compose.yml?

Comment (mfrost) [reply Dockerfile]: I thought about that, but I'm not sure what OS X support is like there. Also, some research suggests that they are not really bind mounts in OS X, and assume the permissions of the host user. Needs testing.

**Comment by djkirby** [comment]
verified user perms looked good for me on os x but it seems to have a problem w/ the migrations

Screen Shot 2019-05-15 at 10 12 39 AM

**Comment by mfrost** [comment]
@djkirby you can bring the database up to speed with `docker-compose exec web rails db:migrate` if the container is already running.

**Comment by cbarber** [comment]
Oooohhhh. That's my bad @djkirby. I thought `rake db:migrate` was still executed via `docker-compose up` but I didn't RTF-PR :)

File: docker-compose.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ version: '3'
 services:
   web:
     build: .
-    command: bash -lc "(rm -rf tmp/pids/ || true) && yarn install && bundle install --path vendor/bundle && bin/rails db:migrate db:seed RAILS_ENV=development && bundle exec foreman start -f Procfile.development"
+    command: bash -lc "(rm -rf tmp/pids/ || true) && yarn install && bundle exec foreman start -f Procfile.development"
```
Comment (chewnoill) [inline docker-compose.yml]: why are we removing the migration from here?

**Comment by aaddaamm** [comment]
@cbarber @mfrost has there been any traction on this PR? is it waiting on something?

File: docker-compose.yml (hunk 2 above)
Comment (mfrost) [reply docker-compose.yml]: I believe there are cases where you might want to boot the app without always running migrations. They are easily executed with `docker-compose exec` so I removed them here.

**Pull request merged**